core/scripts/vrfv2/testnet/main.go

```diff
@@ -646,18 +622,15 @@ func main() {
 		rid, err := consumer.SRequestId(nil)
 		helpers.PanicErr(err)
 		fmt.Printf("Request config %+v Rw %+v Rid %+v\n", rc, rw, rid)
+	case "deploy-bhs-coordinator-consumer":
```

Comment:
Lets call this "deploy-universe" for simplicity. Make sure it also matches the name of the command in the function being called so that `--help` prints the right command name.